src/api/app/mixins/webui/manage_relationships.rb

```diff
@@ -14,11 +14,7 @@ def save_person
     rescue NotFoundError,
            Relationship::SaveError => e
       flash[:error] = e.to_s
-      if params[:webui2]
-        redirect_to users_path
-      else
-        redirect_to add_path(:add_person)
-      end
+      redirect_to add_path(:add_person)
```

Comment:
It's like this now because package and project both have a Bootstrap version, right?

Reply:
> It's like this now because package and project both have a Bootstrap version, right?

Yes.